Testscripts/Linux/utils.sh

```diff
@@ -3730,4 +3730,35 @@ function get_bootconfig_path() {
 		config_path="/usr/boot/config-$(uname -r)"
 	fi
 	echo "$config_path"
+}
+
+function Check_lsvmbus()
+{
+	# check if lsvmbus exists, or the running kernel does not match installed version of linux-tools
+	lsvmbus_path=$(which lsvmbus)
+	if [[ -z "$lsvmbus_path" ]] || ! $lsvmbus_path > /dev/null 2>&1; then
+		install_package wget
+		wget https://raw.githubusercontent.com/torvalds/linux/master/tools/hv/lsvmbus
+		chmod +x lsvmbus
+		if [[ "$DISTRO" =~ "coreos" ]]; then
+			export PATH=$PATH:/usr/share/oem/python/bin/
+			lsvmbus_path="./lsvmbus"
+		else
+			mv lsvmbus /usr/sbin
+			lsvmbus_path=$(which lsvmbus)
+		fi
+	fi
+
+	if [ -z "$lsvmbus_path" ]; then
+		LogErr "lsvmbus tool not found!"
+		SetTestStateFailed
+		exit 0
+	fi
+
+	# lsvmbus requires python
+	which python || [ -f /usr/libexec/platform-python ] && ln -s /usr/libexec/platform-python /sbin/python || which python3 && ln -s $(which python3) /sbin/python
+	if ! which python; then
+		update_repos
+		install_package python
```

Comment:
Is there any chance we fail to install python? I think we had the distro repo access issue before. How do we handle it?

Reply:
If the install_package python failed, the script will be exited.

Reply:
If I remember, the exit is install_package call not the above level. Will see it from the next rum-time.